src/content/learn/responding-to-events.md

```diff
@@ -33,12 +42,15 @@ export default function Button() {
 ```
 
 </Sandpack>
+<div dir="rtl">
+
+با دنبال کردن این سه مرحله می توانید کاری کنید که وقتی کاربر کلیک می کند پیامی نشان دهد:
 
-You can make it show a message when a user clicks by following these three steps:
+1. تابعی به نام `handleClick` *در داخل* کامپوننت `Button` خود تعریف کنید.
+2. منطق را در داخل آن تابع پیاده سازی کنید (از `alert` برای نمایش پیام استفاده کنید).
+3. `onClick={handleClick}` را به JSX المنت `<button>` اضافه کنید.
```

Comment:
is the Right to left showing correctly?

Reply:
yesss. do you think it has a problem?

Reply:
if you saw in the vs code, I don't think so. can you please attach a screen shot if possible?

Reply:
in vs code, it is in this way: 
<img width="1269" alt="Screenshot 1402-05-30 at 09 19 09" src="https://github.com/reactjs/fa.react.dev/assets/72304600/fcb0ebcd-b91f-4ca0-add5-35fc00c7ac97">
but in my localhost, it shows right:
<img width="747" alt="Screenshot 1402-05-30 at 09 18 01" src="https://github.com/reactjs/fa.react.dev/assets/72304600/bc30bd2c-36f8-490f-b105-85dd6afe117c">

what should I do now?